src/Stripe.net/Services/Invoices/InvoiceLineItemAutomaticTaxOptions.cs

```diff
@@ -0,0 +1,14 @@
+// File generated from our OpenAPI spec
+namespace Stripe
+{
+    using Newtonsoft.Json;
+
+    public class InvoiceLineItemAutomaticTaxOptions : INestedOptions
+    {
+        /// <summary>
+        /// Controls whether Stripe will automatically compute tax on this invoice.
+        /// </summary>
+        [JsonProperty("enabled")]
+        public bool? Enabled { get; set; }
+    }
+}
```

Comment:
Not entirely sure what this change is about. Is it just to paginate line items on the upcoming invoice specifically?